saleor/graphql/webhook/subscription_types.py

```diff
@@ -57,6 +57,41 @@ class CategoryDeleted(ObjectType, CategoryBase):
     ...
 
 
+class ChannelBase(AbstractType):
+    channel = graphene.Field(
+        "saleor.graphql.channel.types.Channel",
+        description=f"{ADDED_IN_32} Look up a channel. {PREVIEW_FEATURE}",
+    )
+
+    @staticmethod
+    def resolve_channel(root, info):
+        _, channel = root
+        return channel
+
+
+class ChannelCreated(ObjectType, ChannelBase):
+    ...
+
+
+class ChannelUpdated(ObjectType, ChannelBase):
+    ...
+
+
+class ChannelDeleted(ObjectType, ChannelBase):
+    ...
+
+
+class ChannelStatusChanged(ObjectType, ChannelBase):
+    status = graphene.String(
+        description="New status of a channel.",
+    )
+
+    @staticmethod
+    def resolve_status(root, info):
```

Comment:
What do we achieve by adding this field? Looks like `channel.status` returns ACTIVE/INACTIVE based on value `isActive`. The same result I can achieve by fetching `channel.isActive`, right?

Reply:
I thought that this will be more descriptive but now I think that you are probably right and `isActive` should be enough.